web/src/lib/components/photos-page/memory-lane.svelte

```diff
@@ -36,7 +36,7 @@
   <section
     id="memory-lane"
     bind:this={memoryLaneElement}
-    class="relative mt-5 overflow-x-hidden whitespace-nowrap transition-all"
+    class=" mt-5 overflow-x-hidden whitespace-nowrap transition-all"
```

Comment:
Why do we remove this class here?

Reply:
Forgot to revert as part of cleanup - will add back.